pkg/forwarder/forwarder.go

```diff
@@ -199,12 +200,13 @@ func NewOptions(keysPerDomain map[string][]string) *Options {
 	}
 
 	return &Options{
-		NumberOfWorkers:          config.Datadog.GetInt("forwarder_num_workers"),
-		RetryQueueSize:           config.Datadog.GetInt("forwarder_retry_queue_max_size"),
-		DisableAPIKeyChecking:    false,
-		APIKeyValidationInterval: time.Duration(validationInterval) * time.Minute,
-		KeysPerDomain:            keysPerDomain,
-		ConnectionResetInterval:  time.Duration(config.Datadog.GetInt("forwarder_connection_reset_interval")) * time.Second,
+		NumberOfWorkers:                config.Datadog.GetInt("forwarder_num_workers"),
+		RetryQueueSize:                 config.Datadog.GetInt("forwarder_retry_queue_max_size"),
+		RetryQueuePayloadsTotalMaxSize: config.Datadog.GetInt("forwarder_retry_queue_all_payloads_max_size"),
```

Comment:
I think we should print a deprecated warning when `forwarder_retry_queue_all_payloads_max_size` is used and not `forwarder_retry_queue_max_size`

Reply:
Warning added.